api/src/main/java/io/opentelemetry/metrics/CounterDouble.java

```diff
@@ -29,7 +29,7 @@
  * <pre>{@code
  * class YourClass {
  *
- *   private static final Meter meter = OpenTelemetry.getMeter();
+ *   private static final Meter meter = OpenTelemetry.getMeterFactory().get("some_counter_double");
```

Comment:
use "my_library_name" as example, and everywhere.

Reply:
sure thing!

Reply:
Maybe something like `com.example.MyLibraryName` would be better to indicate that a Java package name is expected here.

Reply:
yeah that seems very helpful!